libevm/VMConfig.h

```diff
@@ -22,17 +22,28 @@ namespace eth
 
 ///////////////////////////////////////////////////////////////////////////////
 //
-// interpreter configuration macros for optimizations and tracing
+// interpreter configuration macros for development, optimizations and tracing
+//
+// EIP_615                - subroutines and static jumps
+// EIP_616                - SIMD
 //
 // EVM_SWITCH_DISPATCH    - dispatch via loop and switch
 // EVM_JUMP_DISPATCH      - dispatch via a jump table - available only on GCC
 //
-// EVM_USE_CONSTANT_POOL  - 256 constants unpacked and ready to assign to stack
+// EVM_USE_CONSTANT_POOL  - constants unpacked and ready to assign to stack
 //
-// EVM_REPLACE_CONST_JUMP - with pre-verified jumps to save runtime lookup
+// EVM_REPLACE_CONST_JUMP - pre-verified jumps to save runtime lookup
 //
 // EVM_TRACE              - provides various levels of tracing
 
+#ifndef EIP_615
+	#define EIP_615 false
```

Comment:
I would prefer defining these in .cmake files instead.
Then we'd be able to build with or without it just by changing cmake command line.

Or maybe it's not even necessary to modify .cmake files. maybe it would be enough just to delete these #defines, then try to build with `cmake .. -DEIP_615=true` ?